### ISSUE TITLE
Plasma Desktop Effects settings don't show preview videos

### ISSUE THREAD

**Issue by TraceyC77** [getsolus/packages#3366]
### Please confirm there isn't an existing open bug report

- [X] I have searched open bugs for this issue

### Summary

In Plasma under Settings - Window Management - Desktop Effects, many effects have preview videos. They aren't loading.

### Steps to reproduce

1. Go to Settings - Window Management - Desktop Effects
2. Click on an effect that has the video icon (e.g. Magic Lamp or Squash)
3. Click the video icon

### Expected result

A preview of the effect plays

### Actual result

There is something spinning that looks like tiny text

![image](https://github.com/user-attachments/assets/1f1a22b3-e441-401c-991f-c3f2a08cfc60)

### Environment

- [X] Is system up to date?

### Repo

Unstable

### Desktop Environment

Plasma

### System details

Operating System: Solus 4.5
KDE Plasma Version: 6.1.3
KDE Frameworks Version: 6.4.0
Qt Version: 6.7.2

### Other comments

I found [a bug in the KDE tracker](https://bugs.kde.org/show_bug.cgi?id=440244) for another distro where the developers indicated this is a packaging issue.

> The needed plugins are:
- gstreamer-plugins-curl for fetching the video data via HTTPS.
- gstreamer-plugins-openh264 to actually play it.

**Comment by ReillyBrogan** [comment]
Do you have `gstreamer-1.0-plugins-bad` installed?

**Comment by ReillyBrogan** [comment]
I looked into this, and I'm pretty sure it's actually an upstream issue:
- This uses qtmultimedia which has used ffmpeg as the default provider since Qt6. We build it with both backends (ffmpeg and gstreamer) but ffmpeg is the default. There is no change in behavior when forcing it to use the gstreamer backend instead
- `Video.qml` (which loads the video player) appears to be using properties that were removed in Qt6, so I suspect this hasn't worked since Plasma 6.0
- Patching it to point to a different type of video file seems to change the error message that it prints out, so it doesn't appear to be a library issue.
